lib/comgr/CMakeLists.txt

```diff
@@ -22,6 +22,14 @@ set(PROJECT_VERSION_PATCH "0")
 set(PROJECT_VERSION
   "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
 
+set(LIB_VERSION_MAJOR ${PROJECT_VERSION_MAJOR})
+set(LIB_VERSION_MINOR ${PROJECT_VERSION_MINOR})
+if ( ${ROCM_PATCH_VERSION} )
```

Comment:
I would prefer `if (DEFINED ROCM_PATCH_VERSION)` here, unless you intend for falsey values in `ROCM_PATCH_VERSION` to be ignored (i.e. if the build server sets `ROCM_PATCH_VERSION=0` it will be ignored).

Reply:
ROCM_PATCH_VERSION is not suppose to be 0, so I can change to DEFINED ROCM_PATCH_VERSION